sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -1088,6 +1142,30 @@ def create_operation(self,
         transform_proto.spec.payload, parameter_type)
     return creator(self, transform_id, transform_proto, payload, consumers)
 
+  def get_timer_coders(self):
+    timer_coder = {}
+    for transform_id, transform_proto in self.descriptor.transforms.items():
```

Comment:
I see us doing this loop three times now. Perhaps it would be more useful to do the loop once to set everything up, creating a single dictionary (transform_id, timer_family_id) -> (all info about that timer we need to dispatch them).